forui/lib/src/widgets/tile/tile_group.dart

```diff
@@ -1,61 +1,128 @@
 import 'package:flutter/foundation.dart';
 import 'package:flutter/widgets.dart';
 
-import 'package:forui/forui.dart';
-import 'package:forui/src/foundation/util.dart';
+import 'package:meta/meta.dart';
 
-/// The divider between tiles in a group.
-enum FTileDivider {
-  /// Represents a divider that spans the entire tile horizontally.
-  full,
+import 'package:forui/forui.dart';
+import 'package:forui/src/foundation/util.dart' as util;
 
-  /// Represents a divider that partially spans the tile horizontally. A tile is always responsible for the divider
-  /// directly below it.
-  ///
-  /// For [FTile.new], the divider spans from the title's left edge to the tile's right edge. It is always aligned to
-  /// the title of the tile above the divider.
-  /// ```
-  /// -----------------------------
-  /// | [prefixIcon] [title]      | <- Tile A
-  /// |              ------------ |
-  /// | [title]                   | <- Tile B
-  /// -----------------------------
-  /// ```
-  indented,
-
-  /// No divider between tiles.
-  none,
-}
+/// A marker interface which denotes that mixed-in widgets can group tiles and be used in a [_MergeTileGroups].
+mixin FTileGroupMixin<T extends Widget> on Widget {}
 
-/// A tile group that groups multiple [FTile]s.
+/// A tile group that groups multiple [FTileMixin]s and [FTileGroupMixin]s together.
 ///
 /// Tiles grouped together will be separated by a divider, specified by [divider].
 ///
 /// See:
 /// * https://forui.dev/docs/tile/tile-group for working examples.
-/// * [FTileGroup] for grouping tiles together.
-/// * [FTileStyle] for customizing a tile's appearance.
-class FTileGroup extends StatelessWidget {
+/// * [FTileGroupStyle] for customizing a tile's appearance.
+class FTileGroup extends StatelessWidget with FTileGroupMixin<FTileMixin> {
   /// The style.
   final FTileGroupStyle? style;
 
-  /// The divider between tiles in a group.
+  /// The divider between tiles. Defaults tp [FTileDivider.indented].
   final FTileDivider divider;
 
+  /// The group's label. It is ignored if the group is part of a [FTileGroup]s.
+  final Widget? label;
+
   /// The group's semantic label.
   final String? semanticLabel;
 
-  /// The group's label.
-  final Widget? label;
-
   /// The tiles in the group.
-  final List<Widget> children;
+  final List<FTileMixin> children;
```

Comment:
😨 So we decided to keep it as a `FTileMixin`?

Reply:
Yup I think it's better to keep it FTileMixin. I accidentally passed in a FTileGroup to the constructor while refactoring the code which kinda sealed it.